### DISCUSSION TITLE
Another noob question: importing hyrule from outside hyrule dir

### DISCUSSION THREAD

**Discussion by Pinjontall94** [hylang/hy#2148] [Q&A]
Hi, I'm super green to this whole thing, but how do you import hyrule macros into a folder _other_ than the downloaded hyrule repo itself? As I understand, I'd normally just use a "from x import x.y" statement, but I'm not sure how that works in Hy. Any help would be very much appreciated!

**Comment by Kodiologist** [comment]
See https://raw.githubusercontent.com/hylang/hy/949e07b4abf639f911dfbda779d2eab2b2ecb62a/docs/api.rst line 599, or https://docs.hylang.org/en/master/api.html#import in the current generated documentation.

**Comment by Pinjontall94** [comment]
Oh, thanks for responding, @Kodiologist! Importing works, but it looks like the only way to do so is if the hyrule repo is in the same directory as my hy code. Is that what you do, or do you append the python path? Neither of those seem idiomatic for python or hy, so I figured I was probably doing something weird.

**Reply by Kodiologist** [reply]
> Importing works, but it looks like the only way to do so is if the hyrule repo is in the same directory as my hy code.

Are you trying to say that `(import hyrule)` raises an error if your current working directory isn't `hyrule`? In that case, it sounds like the package isn't installed correctly. What is the error message? How did you try to install it?
